src/cosalib/cmdlib.py

```diff
@@ -187,10 +187,11 @@ def import_ostree_commit(repo, commit, tarfile):
     if (subprocess.call(['ostree', 'show', '--repo', repo, commit],
                         stdout=subprocess.DEVNULL,
                         stderr=subprocess.DEVNULL) == 0
-            and not os.path.isfile(commitpartial)):
+            and not os.path.isfile(commitpartial)
+            and not force):
         return
 
-    with tempfile.TemporaryDirectory(dir=f'{repo}/tmp') as d:
+    with tempfile.TemporaryDirectory(dir=f'{repo}') as d:
```

Comment:
I think this is a mistake.. We don't want to untar the tarball directly into the repo.

Reply:
OK, I've split that out into https://github.com/coreos/coreos-assembler/pull/806 with more rationale in the commit message.